packages/api/src/routes/taskRoute.js

```diff
@@ -250,4 +250,6 @@ router.get(
   taskController.getHarvestUsesByFarmId,
 );
 
+router.get('/get_field_work_types/:farm_id', taskController.getFieldWorkTypes);
```

Comment:
@anant00729 

Should this not check farm access, or scope or anything?

Reply:
added the scope 
```
router.get(
  '/get_field_work_types/:farm_id',
  hasFarmAccess({ params: 'farm_id' }),
  taskController.getFieldWorkTypes,
);

```